include/sequence.inc

```diff
@@ -0,0 +1,182 @@
+# Hardcoded Instruments
+.define FONTANY_INSTR_SFX      126
+.define FONTANY_INSTR_DRUM     127
+.define FONTANY_INSTR_SAWTOOTH 128
+.define FONTANY_INSTR_TRIANGLE 129
+.define FONTANY_INSTR_SINE     130
+.define FONTANY_INSTR_SQUARE   131
+.define FONTANY_INSTR_NOISE    132
+.define FONTANY_INSTR_BELL     133
+.define FONTANY_INSTR_8PULSE   134
+.define FONTANY_INSTR_4PULSE   135
+.define FONTANY_INSTR_4PULSE2  136
```

Comment:
```suggestion
.define FONTANY_INSTR_ASM_NOISE  136
```
We worked out that this is also random noise that's sampled by randomly reading asm instructions and interpreting it as sampleData:
https://github.com/zeldaret/oot/blob/2e6279bc8e528833704bc6b77635f0eeae6bbe44/src/code/code_800E4FE0.c#L172

Discord discussion on it:
https://discord.com/channels/688807550715560050/688851451014938642/966297685361577984